examples/knative/powercli/kn-pcli-tag/README.md

```diff
@@ -0,0 +1,127 @@
+# kn-pcli-tag
+Example Knative PowerCLI function for applying a vSphere Tag when a Virtual Machine powers on.
+
+# Step 1 - Build
+
+Create the container image locally to test your function logic.
+
+```
+docker build -t <docker-username>/kn-pcli-tag:1.0 .
+```
+
+# Step 2 - Test
+
+Verify the container image works by executing it locally.
+
+Change into the `test` directory
+```console
+cd test
+```
+
+Update the following variable names within the `docker-test-env-variable` file
+
+* VCENTER_SERVER - IP Address or FQDN of the vCenter Server to connect to for vSphere Tagging
+* VCENTER_USERNAME - vCenter account with permission to apply vSphere Tagging
+* VCENTER_PASSWORD - vCenter credentials to account with permission to apply vSphere Tagging
+* VCENTER_TAG_NAME - Name of the vSphere Tag
+* VCENTER_CERTIFCATE_ACTION - Set-PowerCLIConfiguration Action to configure when connection fails due to certificate error (values: Fail, Ignore or Warn)
```

Comment:
what is the default? (that you'll set in secret.json, also see comment below)

Reply:
`Fail` is the default. Will update description to be more clearer